data/json/recipes/food/offal_dishes.json

```
@@ -445,11 +445,22 @@
     "//": "Charcuterie: The Craft of Salting, Smoking, and Curing would be a good book for someone to use for further meat products recipes.",
     "book_learn": [ [ "offalcooking", 3 ] ],
     "qualities": [ { "id": "COOK", "level": 2 }, { "id": "CUT_FINE", "level": 1 } ],
-    "components": [
-      [ [ "meat_stomach", 1, "LIST" ], [ "bag_plastic", 1 ] ],
-      [ [ "salt", 20 ] ],
-      [ [ "water", 2 ], [ "water_clean", 2 ] ]
-    ]
+    "components": [ [ [ "meat_stomach", 1, "LIST" ] ], [ [ "salt", 20 ] ], [ [ "water", 2 ], [ "water_clean", 2 ] ] ]
+  },
+  {
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "result": "sausage_casings_artificial",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_MEAT",
+    "skill_used": "cooking",
+    "difficulty": 5,
+    "time": "50 m",
+    "batch_time_factors": [ 83, 5 ],
+    "charges": 60,
+    "book_learn": [ [ "offalcooking", 3 ] ],
+    "qualities": [ { "id": "COOK", "level": 2 }, { "id": "CUT_FINE", "level": 1 } ],
+    "components": [ [ [ "bag_plastic", 1 ] ], [ [ "salt", 20 ] ], [ [ "water", 2 ], [ "water_clean", 2 ] ] ]
```

Do we need the salt and water? I think that's for preparing the stomach.
```suggestion
    "components": [ [ [ "bag_plastic", 1 ] ] ]
```